exalearn-api/ExaLearn.Dal/Interfaces/IAudioFileRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ExaLearn.Dal.Model;
+
+namespace ExaLearn.Dal.Interfaces
```

Comment:
again something strange empty files without any!!! comments WHY?????????????????

Reply:
Yes, this implementation implies inheritance from the generic repository. these interfaces are implemented so that each one can be specified separately without using a generic wrapper each time.

In this case, the implementation of generic repostiory implies the implementation of the classic crud, and additional functionality can be added to the interfaces as needed.

Reply:
p.s and each interface inherits generic interface

Reply:
in this case, in DI, everything will look like this without a generic wrapper

services.AddScoped <IAudioFileRepository, AudioFileRepository> ();
services.AddScoped <IAudioFileService, AudioFileService> ();
services.AddScoped <IUserRepository, UserRepository> ();
services.AddScoped <IUserService, UserService> ();